versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/Condition.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.store.rocksdb;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A set of conditions that are asserted against an Entity.
+ */
+public class Condition {
```

Comment:
Shouldn't have public classes unless there's a reason to do so.

Reply:
Removed public to make it package local.